src/__test__/utils/processUpload.test.js

```diff
@@ -23,32 +23,35 @@ const validFilesList = [
       name: 'features.tsv.gz',
       path: '/WT13/features.tsv.gz',
       type: 'application/gzip',
-      valid: true,
     },
     upload: { status: UploadStatus.UPLOADING },
     errors: '',
+    compressed: true,
+    valid: true,
   },
   {
     name: 'WT13/barcodes.tsv.gz',
     bundle: {
-      name: 'features.tsv.gz',
+      name: 'barcodes.tsv.gz',
       path: '/WT13/barcodes.tsv.gz',
       type: 'application/gzip',
-      valid: true,
     },
     upload: { status: UploadStatus.UPLOADING },
     errors: '',
+    compressed: true,
+    valid: true,
   },
   {
     name: 'WT13/matrix.mtx.gz',
     bundle: {
-      name: 'features.tsv.gz',
```

Comment:
what was this?

Reply:
The filename in the `bundle.name` property didn't match the outer `name` property. It was simply a copy-paste typo when the tests were written.